classes/class_@gdscript.rst

```diff
@@ -1149,8 +1149,13 @@ Returns the position of the first non-zero digit, after the decimal point. Note
 
 - :ref:`float<class_float>` **stepify** **(** :ref:`float<class_float>` s, :ref:`float<class_float>` step **)**
 
-Snaps float value ``s`` to a given ``step``.
+Snaps float value ``s`` to a given ``step``. This can also be used to round a floating point number to an arbitrary number of decimals.
```

Comment:
This is not "also", you're rephrasing what "Snap float value ``s`` to a given ``step``" means :)

Reply:
I think it will also truncate and not round. But the sentence can likely be made easier to understand, and adding an example seems like a good idea.

Reply:
> This is not "also", you're rephrasing what "Snap float value s to a given step" means :)

Sorry, I misread the example and misunderstood the original description it seems. I didn't know this method could be used to the same effect as `fmod` to snap to a step (as described). I always saw it only used for adjusting the number of decimals.
So your addition seems good :)

Reply:
> I think it will also truncate and not round.

And I tried it and it does round... sorry for commenting before testing :)